components/authz-service/server/v2/policy_test.go

```diff
@@ -467,6 +469,159 @@ func TestCreatePolicy(t *testing.T) {
 	}
 }
 
+func TestCreatePolicy2p1(t *testing.T) {
```

Comment:
Add some explicit 2.1 tests where it matters: create and update with no projects.